codeHF/workflows.yml

```diff
@@ -228,6 +228,11 @@ workflows:
     <<: *taskddbar
     requires_mc: yes
 
+  o2-analysis-hf-task-correlation-d0-hadrons:
+    executable: o2-analysis-hf-task-correlation-d0-hadrons
+    dependencies:
+      - o2-analysis-hf-correlator-d0-hadrons
```

Comment:
Setting `executable` is not necessary if it has the same name as the node.
Keep the dependency on a single line.